packages/nimble-components/src/icon-base/tests/icon-metadata.ts

```diff
@@ -103,9 +106,15 @@ export const iconMetadata: {
     IconBellAndComment: {
         tags: ['alarm', 'notification']
     },
+    IconBellCheck: {
+        tags: ['acknowledged', 'alarm']
+    },
     IconBellCircle: {
         tags: ['alarm']
     },
+    IconBellRadialLines: {
```

Comment:
https://fontawesome.com/icons/bell-on?f=classic&s=solid

Reply:
Font awesome naming for the win 🏆